images/linux/Ubuntu1804-Readme.md

```diff
@@ -17,155 +14,164 @@
-- Docker Compose v2 2.6.1+azure-1
-- Docker-Buildx 0.8.2
-- Docker-Moby Client 20.10.17+azure-1
-- Docker-Moby Server 20.10.17+azure-1
-- Git 2.37.1 (apt source repository: ppa:git-core/ppa)
+- Docker Compose v2 2.11.0+azure-1
+- Docker-Buildx 0.9.1
+- Docker-Moby Client 20.10.18+azure-1
+- Docker-Moby Server 20.10.18+azure-1
+- Git 2.37.3 (apt source repository: ppa:git-core/ppa)
 - Git LFS 3.2.0 (apt source repository: https://packagecloud.io/install/repositories/github/git-lfs)
 - Git-ftp 1.3.1
 - Haveged 1.9.1
-- Heroku 7.60.2
+- Heroku 7.63.4
 - HHVM (HipHop VM) 4.164.0
 - jq 1.5
-- Kind 0.14.0
-- Kubectl 1.24.3
-- Kustomize 4.5.6
-- Leiningen 2.9.8
+- Kind 0.15.0
+- Kubectl 1.25.2
+- Kustomize 4.5.7
+- Leiningen 2.9.10
 - MediaInfo 17.12
 - Mercurial 4.5.3
-- Minikube 1.26.0
+- Minikube 1.27.0
 - n 9.0.0
 - Newman 5.3.2
 - nvm 0.39.1
 - OpenSSL 1.1.1-1ubuntu2.1~18.04.20
-- Packer 1.8.2
-- Parcel 2.6.2
+- Packer 1.8.3
+- Parcel 2.7.0
 - PhantomJS 2.1.1
 - Podman 3.4.2 (apt source repository: https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable)
-- Pulumi 3.37.1
-- R 4.2.0
+- Pulumi 3.40.1
+- R 4.2.1
 - Skopeo 1.5.0 (apt source repository: https://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable)
 - Sphinx Open Source Search Server 2.2.11
 - SVN 1.9.7
-- Terraform 1.2.6
-- yamllint 1.27.1
-- yq 4.26.1
+- Terraform 1.3.0
+- yamllint 1.28.0
+- yq 4.27.5
 - zstd 1.5.2 (homebrew)
 
 ### CLI Tools
-- Alibaba Cloud CLI 3.0.124
-- AWS CLI 2.7.20
+- Alibaba Cloud CLI 3.0.127
+- AWS CLI 2.7.34
 - AWS CLI Session manager plugin 1.2.339.0
-- AWS SAM CLI 1.53.0
-- Azure CLI (azure-cli) 2.38.0 (installation method: https://docs.microsoft.com/en-us/cli/azure/install-azure-cli-linux?pivots=apt)
+- AWS SAM CLI 1.57.0
+- Azure CLI (azure-cli) 2.40.0 (installation method: https://docs.microsoft.com/en-us/cli/azure/install-azure-cli-linux?pivots=apt)
 - Azure CLI (azure-devops) 0.25.0
-- GitHub CLI 2.14.3
-- Google Cloud SDK 369.0.0 (apt source repository: https://packages.cloud.google.com/apt)
+- GitHub CLI 2.16.0
+- Google Cloud SDK 403.0.0 (apt source repository: https://packages.cloud.google.com/apt)
 - Hub CLI 2.14.2
-- Netlify CLI 10.13.0
-- OpenShift CLI 4.10.24
-- ORAS CLI 0.13.0
-- Vercel CLI 27.3.3
+- Netlify CLI 11.8.2
+- OpenShift CLI 4.11.5
+- ORAS CLI 0.14.1
+- Vercel CLI 28.4.1
 
 ### Java
 | Version             | Vendor          | Environment Variable |
 | ------------------- | --------------- | -------------------- |
-| 8.0.332+9 (default) | Eclipse Temurin | JAVA_HOME_8_X64      |
-| 11.0.16+8           | Eclipse Temurin | JAVA_HOME_11_X64     |
+| 8.0.345+1 (default) | Eclipse Temurin | JAVA_HOME_8_X64      |
+| 11.0.16+1           | Eclipse Temurin | JAVA_HOME_11_X64     |
 | 12.0.2+10           | Adopt OpenJDK   | JAVA_HOME_12_X64     |
-| 17.0.4+8            | Eclipse Temurin | JAVA_HOME_17_X64     |
+| 17.0.4+1            | Eclipse Temurin | JAVA_HOME_17_X64     |
```

Comment:
@mikhailkoliada  is this part ok?

Reply:
It's normal: temurin-17-jdk_17.0.4.0.0+8 -> temurin-17-jdk_17.0.4.1.0+1